test/base/EstimateGasEscrowClaimTokensTest.js

```diff
@@ -17,8 +17,8 @@ const Escrow = artifacts.require("./mock/base/EscrowMock.sol");
 contract("EstimateGasEscrowClaimTokensTest", function(accounts) {
   const loansEncoder = new LoansBaseInterfaceEncoder(web3);
   
-  const baseGasCost = 452022; // Gas cost with 1 token in wallet
-  const expectedGasCost = (tokens) => baseGasCost + ((tokens -  1) * 37000); // Gas cost > 1 token in wallet
+  const baseGasCost = 453453; // Gas cost with 1 token in wallet
+  const expectedGasCost = (tokens) => baseGasCost + ((tokens -  1) * 167151); // Gas cost > 1 token in wallet
```

Comment:
this is quite a hike in expected gas cost @tpscrpt @sypeer

Reply:
Yeah I'll review this again and try to find why, currently my best guess is just the added bytes(32) read from storage on every getLoan() call (since it's kept in another contract and can't just extract 1 value). Might have to refactor so this loan gets passed around instead of read multiple times from storage.